core/src/test/scala/org/apache/spark/benchmark/Benchmark.scala

```diff
@@ -116,14 +118,15 @@ private[spark] class Benchmark(
       name + ":", "Best Time(ms)", "Avg Time(ms)", "Stdev(ms)", "Rate(M/s)", "Per Row(ns)", "Relative")
```

Comment:
I believe the benchmark result should be self-readable. A reader shouldn't consult to source code to figure out meaning of `Relative`. How about to indicate that in the header.